### DISCUSSION TITLE
MS Store app 💀

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [CommunityToolkit/Windows#272] [Feedback]
Hey, guys? Are you the people that should own the MS app "Windows Community Toolkit Gallery" https://www.microsoft.com/store/productId/9NBLGGH4TLCQ?ocid=pdpshare ? That thing hasn't been updated since 2016 and is pretty useless. If you can find the right people (with the right access) can you either update the app or take it offline? Thanks :)

**Comment by Panda-Sharp** [comment]
@Jay-o-Way actually that app is constantly update, last update was with last release of the WCT in September.

**Comment by niels9001** [comment]
@Jay-o-Way Yes, the app is part of the `tooling` submodule and got updated last month (with another update coming soon).

It was _released_ in 2016, but got updated since then.

![image](https://github.com/CommunityToolkit/Windows/assets/9866362/28a37b50-4dfb-45b4-bced-e78e6ecfb919)

**Reply by Jay-o-Way** [reply]
Ohhh okay then. I don't see any version number and 2016 is the only year I saw. 👍🏻
